### PR TITLE
Move command to a code-block

### PR DESCRIPTION
**Pull request by prasadkatti**
Put the `kubectl apply` command in a separate code block to keep things consistent

### CODE REVIEW

File: content/en/docs/tasks/run-application/configure-pdb.md [hunk 1]
```diff
@@ -173,7 +173,10 @@ automatically responds to changes in the number of replicas of the corresponding
 
 ## Create the PDB object
 
-You can create or update the PDB object with a command like `kubectl apply -f mypdb.yaml`.
+You can create or update the PDB object using kubectl.
+```shell
```
Comment (divya-mohan0209) [inline content/en/docs/tasks/run-application/configure-pdb.md:177]: To maintain consistency, can we have a newline entry before ```shell

File: content/en/docs/tasks/run-application/configure-pdb.md [hunk 2]
```diff
@@ -173,7 +173,10 @@ automatically responds to changes in the number of replicas of the corresponding
 
 ## Create the PDB object
 
-You can create or update the PDB object with a command like `kubectl apply -f mypdb.yaml`.
+You can create or update the PDB object using kubectl.
```
Comment (divya-mohan0209) [inline content/en/docs/tasks/run-application/configure-pdb.md:176]:
```suggestion
You can create or update the PDB object using kubectl.

```

File: content/en/docs/tasks/run-application/configure-pdb.md (hunk 1 above)
Comment (divya-mohan0209) [reply content/en/docs/tasks/run-application/configure-pdb.md:177]: I've added it as a suggestion above.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
lgtm except adding new line before ```shell for consistency.

**Review by sftim — COMMENTED** [review]
/lgtm

Extra tidying is welcome, but I want to prioritize the main improvement.
/approve

**Pull request merged**